python/paddle/fluid/layers/control_flow.py

```diff
@@ -1018,8 +1018,17 @@ def body(i, ten):
         return loop_vars
 
     while_loop_block = While(pre_cond, is_test, name)
+    with_mutable_vars = assert_with_mutable_vars(loop_vars)
```

Comment:
The name shouldn't be "assert_XX" because assert usually means failure if some conditions don't meet. Please modify to "has_mutable_vars" or similar name

Reply:
Thanks, code is modified into `has_mutable_vars_in_loop = hold_mutable_vars(loop_vars)`